src/pages.ts

```diff
@@ -0,0 +1,3 @@
+// example registering lit component.
+// import MozChat from './features/chat/MozChat'
+// customElements.define('moz-chat', MozChat)
```

Comment:
this feels like it's going to be duplicate of sidebar.ts, so i'll rename and reuse elements.ts